### DISCUSSION TITLE
如果某个任务出现问题无法完成，能否跳过这个任务继续执行还未执行过的任务

### DISCUSSION THREAD

**Discussion by iocmicu** [LmeSzinc/AzurLaneAutoScript#2418] [Q&A]
大舰队物资无法购买，会错误停止无法继续其他的任务
如果遇到错误停止是目前最好的方案，能否将出现错误的环节往后顺延到最后一个：
1.发生无法自动解决的错误
2.将发生的错误的任务放到队列最后
3.继续执行队列其他任务
4.再执行一遍刚才发生错误的任务
5.如果还是错误，再终止运行

**Comment by LmeSzinc** [comment]
https://github.com/LmeSzinc/AzurLaneAutoScript/pull/1033#issuecomment-1213320420

**Reply by iocmicu** [reply]
不好意思表达的不是很好，如果不改变现有的东西，只是在手动启动调度器之后，将全部任务执行一遍之后再报错停止呢？几乎所有界面都有右上角回到主界面的按钮，如果没有那个按钮或者按不了再停止这样会不会更好
